Prace_domowe/Praca_domowa3/Grupa1/Jakub_Foltyn/PD3.ipynb

```diff
@@ -0,0 +1,670 @@
+{
```

Comment:
<p>Boxplot to odpowiedni sposób wizualizacji. Być może lepiej byłoby zrobić trzy oddzielne wykresy, tak aby każdy był dedykowany innej mierze.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>